src/openmrs-backend-dependencies.ts

```diff
@@ -1,4 +1,4 @@
 export const backendDependencies = {
   "webservices.rest": "^2.24.0",
-  fhir: "1.18.0"
+  fhir: "^1.18.0"
```

Comment:
The `^` allows for future versions of fhir to be fine for esm-primary-navigation.